### PR TITLE
Module ui

### PR DESCRIPTION
**Pull request by Sonith**
Base UI

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -4,9 +4,10 @@
    Author's custom styles
    ========================================================================== */
 
-
-
-
+body {
+   padding-top: 50px;
+   padding-bottom: 20px;
+}
```
Comment (40x) [inline css/main.css:11]: avoid hardcoding the px values, use % instead

File: index.html [hunk 2]
```diff
@@ -1,96 +1,164 @@
...
+<footer class="container">
+    <p>&copy; Company 2015</p>
+</footer>
 
-        <script src="js/main.js"></script>
-    </body>
-</html>
+<!-- /container -->
+<script src="js/vendor/jquery-1.11.2.min.js"></script>
+<script>window.jQuery || document.write('<script src="js/vendor/jquery-1.11.2.min.js"><\/script>')</script>
```
Comment (40x) [inline index.html:160]: why is this needed?

File: owner.html [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <style>
+        body {
+            padding-top: 50px;
```
Comment (40x) [inline owner.html:14]: avoid internal styles

File: owner.html [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+            padding-bottom: 20px;
+        }
+    </style>
+    <link rel="stylesheet" href="css/bootstrap-theme.min.css">
+    <link rel="stylesheet" href="css/main.css">
+
+    <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
```
Comment (40x) [inline owner.html:21]: load scripts at the bottom of body

**Comment by Sonith** [comment]
All those came from the HTML5Boilerplate. I forgot to remove/update those while submitting. I had since removed/updated this. Thanks for the review
